packages/devtools_app/lib/src/screens/app_size/app_size_table.dart

```diff
@@ -11,21 +11,28 @@ import '../../shared/table_data.dart';
 import '../../shared/utils.dart';
 import '../../ui/colors.dart';
 
+import 'app_size_controller.dart';
+
 class AppSizeAnalysisTable extends StatelessWidget {
-  factory AppSizeAnalysisTable({required TreemapNode rootNode}) {
-    final treeColumn = _NameColumn(currentRootLevel: rootNode.level);
+  factory AppSizeAnalysisTable({
+    required TreemapNode rootNode,
+    required AppSizeController controller,
+  }) {
+    final treeColumn =
+        _NameColumn(currentRootLevel: rootNode.children[0].level);
```

Comment:
I'm guessing this is part of the deferred loading changes? Deferred loading-specific logic should be removed from this PR